server/dex/dex.go

```diff
@@ -74,11 +77,11 @@ type DexConf struct {
 	Markets           []*dex.MarketInfo
 	Assets            []*AssetConf
 	Network           dex.Network
+	BondIncrement     uint64
```

Comment:
Need to remove this `BondIncrement` field since bond parameters a per-asset and included in `AssetConf` from markets.json